app/src/main/java/com/hamdan/forzenbook/data/repository/CreateAccountRepository.kt

```diff
@@ -7,5 +7,5 @@ interface CreateAccountRepository {
         birthDay: String,
         email: String,
         location: String,
-    ): Int
+    ): Boolean
```

Comment:
It doesn't make sense for this to return a boolean. This function either succeeds or it throws an exception, either Generic or User Exists. The boolean is redundant

Reply:
Ok so I guess it should really return nothing because an enum doesn't make sense here for a singular option.

Reply:
yea that seems cleaner